docs/masking-providers.md

```diff
@@ -89,13 +89,21 @@ options and their default values.
 >   The interval's width is configurable, but is constant across the domain of values.  The range
 >   is inclusive on the lower bound and exclusive on the upper bound. The format in which the interval is presented is 
 >   configurable.  For examples of valid formats, see the java documentation for java.util.Formatter.
-
-| **Option name**       | **Type** | **Description**                         | **Default value** |
-|-----------------------|----------|-----------------------------------------|-------------------|
-| binSize               | Integer  | The range of the interval               | 5                 |
-| format                | String   | The format of the replacement range     | %s-%s             |
-| startValue            | Integer  | The lower bound of the range            | 0                 |
-| useStartValue         | Boolean  | Use the startValue as the lower bound   | false             |
+>   For threshold values, it uses a constant values. For example if values is 3, its under threshold value would be >10.
+>   Useful for age generalizations: <8, 5-10, 10-15, ..., 85-90, 90+.
```

Comment:
Might want to say something like:  The provider also optionally supports grouping all values below a given threshold and/or all values above a given threshold into a single interval with a given, constant label.  This is useful for age generalization, allowing replacement values such as:
<8, 5-10, 10-15, ..., 85-90, 90+